RELEASE-NOTES.txt

```diff
@@ -1,5 +1,7 @@
-Unreleased
+1.59.0
```

Comment:
Based upon the PR checklist, we should retain the Unreleased heading for future use. 
```suggestion
Unreleased
---

1.59.0
```